database.js

```diff
@@ -5,7 +5,11 @@
  *
 */
 const mysql = require('mysql')
-const config = require('./database_config.json')
+try {
+	const config = require('./database_config.json')
```

Comment:
I think this makes the `config` only scoped to within the try-catch. createConnection is failing for me because config is not defined.